pkg/loop/internal/pb/ccipocr3/msghasher.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb/ccipocr3;ccipocr3pb";
+
+package loop.internal.pb.ccipocr3;
+
+import "models.proto";
+
+service MsgHasher {
```

Comment:
Lets put a comment on what this service will do and how instances of it are implemented/organized (i.e per chain, per chain family, etc.). Note that all comments you put in the proto files are automatically ported to the generated code (in this case Go) so its not wasteful.

e.g

```suggestion
// MsgHasher constructs the message hash of the provided message in a chain-family-specific
// fashion. Thus, there is a different implementation of the MsgHasher for each chain family (e.g,
// EVM, Solana, Starknet, etc.).
service MsgHasher {
```